examples/vweb/vweb_websocket/assets/websocket_client.js

```diff
@@ -0,0 +1,22 @@
+const messageList = document.getElementById('message-list');
+const protocol = location.protocol === 'https:' ? 'wss' : 'ws';
+const socket = new WebSocket(`${protocol}://${location.host}/ws`);
+let i = 0;
+
+function send(message) {
+  messageList.innerHTML += `<li>&gt; ${message}</li>`;
+  socket.send(message);
+}
+
+socket.addEventListener("open", (event) => {
+  console.log('Connected to WS server');
+  send('Salut tout le monde !');
+});
+
+socket.addEventListener("message", (event) => {
+  const { data } = event;
+  messageList.innerHTML += `<li>&lt; ${data}</li>`;
+  setTimeout(() => {
+    send(`Message reçu ${i++}`);
```

Comment:
```v
    send(`Message reçu ${i++}`);
```
There may be a problem with the file character set:   ```reçu```

Reply:
fixed